libbeat/_meta/config/output-elasticsearch.reference.yml.tmpl

```diff
@@ -81,6 +81,14 @@ output.elasticsearch:
   #allow_older_versions: false
 
 {{include "ssl.reference.yml.tmpl" . | indent 2 }}
+
+  # Enable exiting {{.BeatName}} if any of the files pointed by
+  # `ssl.certificate_authorities` changes
+  #ssl.exit_on_cert_change.enabled: false
+
+  # Interval to scan for changes on CA certificate files
+  #ssl.exit_on_cert_change.period: 1m
```

Comment:
Nitpicking but why different words are used? `interval` vs `period`? I'd rather use `interval` everywhere.

Reply:
I use `period` as config key to be consistent with config reload (I'm using the same struct):
https://github.com/elastic/beats/blob/ad651a79276c3570f6077723f5df02bf3ea2da36/filebeat/filebeat.reference.yml#L1249-L1250

But I can update the comment to use "period" instead of "interval".

Reply:
I'm for consistency. In the config docs it's not using the word `interval` though https://www.elastic.co/guide/en/beats/filebeat/current/_live_reloading.html

Double-wording is a bit confusing, it can cause someone using `interval` as the config key instead of `period`.

It's not a strong opinion though.

Reply:
I updated the comment to use "period", I tried to keep it short here because it's the reference doc. I'll push my changes very soon.